### PR TITLE
Instrument class

### PR DESCRIPTION
**Pull request by JunCEEE**
@shervin86 @mads-bertelsen 
As discussed, this instrument class PR is submitted after the approval of the parameters class. 

@mads-bertelsen I guess the PR for the master parameters class will not affect this?

### CODE REVIEW

**Review by shervin86 — APPROVED** [review]

File: libpyvinyl/Instrument.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    def __init__(self, name, calculators=None):
+        """An Instrument class
+
+        :param name: The name of this instrument
+        :type name: str
+        :param calculators: A collection of Calculator objects.
+        :type calculators: list
+        """
+        self.name = name
+        self.calculators = {}
```
Comment (mads-bertelsen) [inline libpyvinyl/Instrument.py:20]: This is a bit confusing, as a user I would see that the calculators input need to take "A collection" of calculator objects, but the type is "list" and the the calculators object is initialised as a "dict".

**Review by mads-bertelsen — APPROVED** [review]
All good! Just fix the one little documentation problem.

I think this neat little class will make life so much easier for users that didn't necessarily write the simulation themselves. Great!

File: libpyvinyl/Instrument.py (hunk 1 above)
Comment (JunCEEE) [reply libpyvinyl/Instrument.py]: Thank you, Mads, well spotted! It was a list at the beginning, and I forgot to update it.

**Review by CFGrote — COMMENTED** [review]
* Please check pep8 (code formatting). e.g empty line at the bottom.

* init(): I would be stricter on having all attributes hidden away as `self.__attribute` and define accessors for getting and setting attribute values, decorated as  `@property` and `setter`, respectively. 

* Docstrings: I always leave a blank line between two `:type ...:` and the next `:param...:` and don't mix google, numpy, and doxygen styles.

**Pull request merged**